Source/MMWormhole.h

```diff
@@ -74,6 +76,22 @@ FOUNDATION_EXPORT const unsigned char MMWormholeVersionString[];
 @interface MMWormhole : NSObject
 
 /**
+ The wormhole messenger is an object that conforms to the MMWormholeTransiting protocol. By default
+ this object will be set to a default implementation of this protocol which handles archiving and
+ unarchiving the message to the shared app group in a file named after the identifier of the 
+ message.
+ 
+ Users of this class may create their own implementation of the MMWormholeTransiting protocol to use
+ for the purpose of defining the means by which messages transit the wormhole. You could use this to
+ change the way that MMWormhole stores messages as files, to read and write messages to a database,
+ or otherwise be notified in other ways when messages are changed.
+ 
+ @warning While changing this property is optional, the value of the wormhole messenger should 
+ not be nil and is required for the class to work.
+ */
+@property (nonatomic, strong) id<MMWormholeTransiting> wormholeMessenger;
```

Comment:
Should we change this to be `@property (nonatomic, strong, nonnull) id<MMWormholeTransiting> wormholeMessenger;`?